databricks_cli/sdk/api_client.py

```diff
@@ -147,16 +144,14 @@ def perform_query(self, method, path, data={}, headers=None, files=None, version
             except ValueError:
                 pass
             raise requests.exceptions.HTTPError(message, response=e.response)
-        if return_raw_response:
-            return resp
         return resp.json()
 
 
     def get_url(self, path, version=None):
         if version:
             return self.url + version + path
         elif self.jobs_api_version and path and path.startswith('/jobs'):
-            return self.url + self.jobs_api_version + path
+            return self.url + self.jobs_api_version + path 
```

Comment:
If this doesn't make a re-revert difficult, we can probably remove the extra whitespace here.